spaces/domains.json

```
@@ -499,6 +499,9 @@
   "vote.jfscaramazza.com": "jfscaramazza.eth",
   "signal.paladin.vote": "palvote.eth",
   "dao.cronosfrens.io": "cronosfrens.io",
+  "vote.tresleches.finance": "3leches.eth",
+  "gov.fxcm.xyz": "symbolspace.eth",
+  "gov.wagumi.xyz": "  wagumi. eth"
```

A comma is missing at the end of the line above your sub-domain.
```suggestion
  "gov.wagumi.xyz": "  wagumi. eth",
```